proxy/apifilterproxy/manager/filter_manager.go

```diff
@@ -0,0 +1,199 @@
+package manager
+
+import (
+	"encoding/json"
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+	"strings"
+
+	"github.com/rancher/websocket-proxy/proxy/apifilterproxy/filters"
+	//to register http filter
+	_ "github.com/rancher/websocket-proxy/proxy/apifilterproxy/filters/http"
+	//to register auth filter
+	_ "github.com/rancher/websocket-proxy/proxy/apifilterproxy/filters/auth"
+	"github.com/rancher/websocket-proxy/proxy/apifilterproxy/model"
+	"github.com/rancher/websocket-proxy/proxy/apifilterproxy/util"
+)
+
+var (
```

Comment:
We should get away from using package level variables like this. Can you refactor this into a struct with properties?